packages/models-library/src/models_library/projects_state.py

```diff
@@ -27,6 +27,17 @@ class RunningState(str, Enum):
     FAILED = "FAILED"
     ABORTED = "ABORTED"
 
+    def is_running(self) -> bool:
+        return self in (
+            RunningState.PUBLISHED,
+            RunningState.PENDING,
+            RunningState.STARTED,
+            RunningState.RETRY,
+        )
+
+    def is_stopped(self) -> bool:
+        return not self.is_running()
```

Comment:
Is it really that necessary? Using `not is_running()` should not be that bad. Or was there a different reason for this?

Reply:
NO, it is not necessary. I just tried to parallel what we have in computations utils